src/clarity-icons/clarity-icons.scss

```diff
@@ -71,6 +83,14 @@ clr-icon {
     &[dir="left"] {
         @include rotateSVGIcon(270deg);
     }
+    &[shape$=" flip-horizontal"],
```

Comment:
I am happy with allowing `flip-horizontal` in the `shape` attribute and just horizontal in the `dir` attribute as @victormejia currently has it. But @mathisscott,  do you think we need to keep the consistency over these orientation names across `shape` and `dir` attributes?